gnocchi/metric/v1/measures/requests.go

```diff
@@ -134,3 +134,74 @@ func Create(client *gophercloud.ServiceClient, metricID string, opts CreateOptsB
 	})
 	return
 }
+
+// BatchCreateMetricsOptsBuilder is needed to add measures to the BatchCreateMetrics request.
+type BatchCreateMetricsOptsBuilder interface {
+	ToBatchCreateMetricsMap() (map[string]interface{}, error)
```

Comment:
This should be `ToMeasuresBatchCreateMetricsMap`. The format of the interface name is `To-ResourcePkg-Action-RequestType`.